css/style.css

```diff
@@ -6,4 +9,40 @@
 .appview-container.st-emotion-cache-1wrcr25.ea3mdgi6 {
   background-color: rgb(122, 195, 231) !important;
   top: 100px !important;
+}
+
```

Comment:
It might be helpful to add some comments on where these stylings are being applied (i.e., which python file and which component exactly), just to make navigation across different code files more intuitive.